source/acquaman/VESPERS/VESPERSXASScanActionController.cpp

```diff
@@ -218,6 +218,12 @@ AMAction3* VESPERSXASScanActionController::createInitializationActions()
 			initializationAction->addSubAction(VESPERSBeamline::vespers()->realSampleStageMotorGroupObject()->horizontalAxis()->createTranslateMoveAction(configuration_->x()));
 			initializationAction->addSubAction(VESPERSBeamline::vespers()->realSampleStageMotorGroupObject()->verticalAxis()->createTranslateMoveAction(configuration_->y()));
 		}
+
+		else if (motor.testFlag(VESPERS::BigBeamX) && motor.testFlag(VESPERS::BigBeamZ)){
+
+			initializationAction->addSubAction(VESPERSBeamline::vespers()->bigBeamMotorGroupObject()->horizontalAxis()->createTranslateMoveAction(configuration_->x()));
+			initializationAction->addSubAction(VESPERSBeamline::vespers()->bigBeamMotorGroupObject()->verticalAxis()->createTranslateMoveAction(configuration_->y()));
```

Comment:
Yeah, it's right.  The "Go To Position" aspect to the XAS scan is VERY old code and uses X and Y.  VESPERS uses the general convention of X and Z for horizontal and vertical motion because Y is reserved for the beam direction.